news/146.bug

```diff
@@ -0,0 +1 @@
+Respect locally allowed types when pasting objects [cekk]
```

Comment:
Late on the party, but this suffix `.bug` will not work with `towncrier` expectations, see https://github.com/plone/plone.dexterity/blob/master/pyproject.toml it should be `bugfix`

I will change it directly on master branch.